config/initializers/omniauth.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET']
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
+    { :scope => 'userinfo.email, userinfo.profile, https://www.googleapis.com/auth/calendar' }
```

Comment:
Figuring out the scope here was tricky. Who knew you needed the userinfo and the calendar scope.